20180908/jobCentTelegramBot/sponsor/server/controllers/sponsors.js

```diff
@@ -0,0 +1,56 @@
+const Sponsor = require("../models").Sponsor;
+const nCentSDK = require("ncent-sandbox-sdk");
+const nCentSDKInstance = new nCentSDK();
+
+module.exports = {
+  create(req, res) {
```

Comment:
I wouldn't recommend using an object like this - you could end up with a bunch of nulls. You should probably use multiple parameters for anything required and make some of them optional. You should also document what each of the parameters is with comments.